tutorials/034 - Distributing Calls using Ray.ipynb

```diff
@@ -0,0 +1,415 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "[![AWS SDK for pandas](_static/logo.png \"AWS SDK for pandas\")](https://github.com/aws/aws-sdk-pandas)\n",
+    "\n",
+    "# 34 - Distributing Calls Using Ray\n",
+    "\n",
+    "AWS SDK for pandas supports distribution of specific calls using [ray](https://docs.ray.io/) and [modin](https://modin.readthedocs.io/en/stable/).\n",
+    "\n",
+    "When this is enabled, data loading methods return Modin data frame instead of Pandas data frames. Modin provides seamless integration and compatibility with existing Pandas code, with the benefit of distributing operations across your Ray instance. The benefit is being able to operate at a much higher scale."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "%pip install \"awswrangler[ray,modin]==3.0.0b2\""
```

Comment:
Minor - need to add this file to `.bump2version.cfg` then

Reply:
I'm not sure we want to do that. Once we release `3.0.0`, we'll change this to just `pip install awswrangler[ray,modin]`, without needing to specify the version. It's what we do in all our other tutorials.

Reply:
Yeah but until then why not ensure the version is up-to-date? Can't hurt. Also we already do this for 034 tutorial and a bunch of others (for the documentation links mainly)

Reply:
Fair point.